pages/api/connectTest.ts

```diff
@@ -0,0 +1,15 @@
+import { dbConnect } from './_dbConnect'
```

Comment:
This file contains a simple test that attempts to connect to the client and tells us whether the connection was successful. This endpoint does not actually serve a purpose in our application, but is rather a 'sanity check' so that we know our client is properly connected to the mongo server/database.